package.json

```diff
@@ -121,6 +121,7 @@
   "scripts": {
     "build": "node ./node_modules/webpack/bin/webpack.js --mode production",
     "build:dev": "node ./node_modules/webpack/bin/webpack.js --mode development",
+    "build:mcp": "find mcp-servers -type d -depth 1 -exec bash -c 'cd {} && yarn build' \\;",
```

Comment:
Could this be made cross-OS compatible? The current format does not work on Windows.